### PR TITLE
(improvement) Offsets healthcheck to T+1m

### PR DESCRIPTION
**Pull request by denzuko**
Changes:
   - Added the `--start-period=5m` option to the docker healthcheck which offsets the mcstatus commands by 5 minutes to allow modpacks to complete downloading.

### CODE REVIEW

**Comment by itzg** [comment]
I'm now worried this change from the default of 0 seconds to 5 minutes might be too disruptive for anyone with a system running vanilla containers and expecting containers to report readiness sooner than 5 minutes.

How about a default of 1 minute and then you can use the override approaches to increase the start period to 5 minutes for FTB, etc containers?

**Comment by denzuko** [comment]
Been testing and finding that the larger packs like FTB (ie 75-125Mb+) take about 25 minutes to start otherwise the healthcheck kills the container before the zip completes.

File: Dockerfile [hunk 1]
```diff
@@ -20,7 +20,8 @@ RUN apk add --no-cache -U \
 
 RUN pip install mcstatus yq
 
-HEALTHCHECK CMD mcstatus localhost:$SERVER_PORT ping
+ARG HEALTHCHECK_DEFAULT_START=5m
```
Comment (itzg) [inline Dockerfile]: Are you good with changing this to `1m` as a compromise for vanilla containers?

Comment (denzuko) [reply Dockerfile]: that should work.

Comment (itzg) [reply Dockerfile]: Cool. When you change that I'll be good to merge this in.

File: Dockerfile [hunk 2]
```diff
@@ -20,8 +20,7 @@ RUN apk add --no-cache -U \
 
 RUN pip install mcstatus yq
 
-ARG HEALTHCHECK_DEFAULT_START=5m
```
Comment (itzg) [inline Dockerfile]: I liked the build arg, I was just wanting a value of `1m` instead of `5m`

Comment (denzuko) [reply Dockerfile]: I liked it too but had a hell of a type trying to get it to work so I just revered to a know good state.

Comment (itzg) [reply Dockerfile]: Oh, I wonder if build args don't carry over to the health check's execution environment.

File: Dockerfile [hunk 3]
```diff
@@ -20,7 +20,7 @@ RUN apk add --no-cache -U \
 
 RUN pip install mcstatus yq
 
-HEALTHCHECK CMD mcstatus localhost:$SERVER_PORT ping
+HEALTHCHECK --start-period=5m CMD mcstatus localhost:$SERVER_PORT ping
```
Comment (itzg) [inline Dockerfile]: I know it conflicts with the PR title :) , but can you change this to 1m for the default:
```suggestion
HEALTHCHECK --start-period=1m CMD mcstatus localhost:$SERVER_PORT ping
```

Comment (denzuko) [reply Dockerfile]: Done.

**Pull request merged**